compile question

The latest merge of paddle had compiling problems on CentOS 6.3.
```
[100%] [100%] Building CXX object paddle/pybind/CMakeFiles/paddle_pybind.dir/pybind.cc.o
Building CXX object paddle/pybind/CMakeFiles/paddle_pybind.dir/exception.cc.o
In file included from /home/xxx/.jumbo/include/python2.7/Python.h:8:0,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/common.h:99,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/pytypes.h:12,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/cast.h:13,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/attr.h:13,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/pybind11.h:36,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/pybind/exception.h:17,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/pybind/exception.cc:15:
/home/xxx/.jumbo/include/python2.7/pyconfig.h:1161:0: error: "_POSIX_C_SOURCE" redefined [-Werror]
 #define _POSIX_C_SOURCE 200112L
 ^
In file included from /usr/include/dlfcn.h:23:0,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/platform/enforce.h:17,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/pybind/exception.h:16,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/pybind/exception.cc:15:
/usr/include/features.h:162:0: note: this is the location of the previous definition
 # define _POSIX_C_SOURCE 200809L
 ^
In file included from /home/xxx/.jumbo/include/python2.7/Python.h:8:0,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/common.h:99,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/pytypes.h:12,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/cast.h:13,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/attr.h:13,
                 from /home/xxx/program/temp_pool_op/Paddle/build/third_party/pybind/src/extern_pybind/include/pybind11/pybind11.h:36,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/pybind/exception.h:17,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/pybind/exception.cc:15:
/home/xxx/.jumbo/include/python2.7/pyconfig.h:1183:0: error: "_XOPEN_SOURCE" redefined [-Werror]
 #define _XOPEN_SOURCE 600
 ^
In file included from /usr/include/dlfcn.h:23:0,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/platform/enforce.h:17,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/pybind/exception.h:16,
                 from /home/xxx/program/temp_pool_op/Paddle/paddle/pybind/exception.cc:15:
/usr/include/features.h:164:0: note: this is the location of the previous definition
 # define _XOPEN_SOURCE 700
 ^
[100%] Building CXX object paddle/api/CMakeFiles/_swig_paddle.dir/PaddlePYTHON_wrap.cxx.o
```

--- a/paddle/pybind/exception.h
+++ b/paddle/pybind/exception.h
@@ -13,6 +13,7 @@
    limitations under the License. */
 
 #pragma once
+#include <Python.h>
 #include "paddle/platform/enforce.h"
 #include "pybind11/pybind11.h"
 namespace paddle {

